model/Licensing/Licensing.md

```diff
@@ -0,0 +1,97 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Licensing
+
+## Summary
+
+The Licensing Profile defines a minimum set of license information required to comply with typical license compliance use cases.
+
+## Description
+
+The Licensing profile only contains additional restrictions.
```

Comment:
What is this meant to be saying?

I probably wouldn't use this phrasing -- "additional restrictions" in the open source licensing context is often used to mean e.g. provisions in license texts that impose additional requirements or prohibitions.

Reply:
I clarified this to be related to the model